PolinaGushcha/Portfolio-NextJs
# Pages/html

# site config, HTML, form validation and email sender func

**What is the context?** 
At this point I have initialized the project, implemented the html part of the site and a couple of interacting parts such as form validation. Subsequently, branches will be divided into major fiches

**What has been changed**
Project structure and configuration is set up. Basic html has been added. Form validation is configured. Justification function of e-mail message is created 

**How to test it?**
You need to enter data into the form to send a message to my email. The phone validation should be 9 digits, later the phone validation will be slightly changed to a more correct one
